### ISSUE TITLE
Document versions of used libraries in the README

### ISSUE THREAD

**Issue by nlohmann** [lovasoa/bad_json_parsers#27]
As the libraries themselves are not part of the repository, it would be really helpful to see which version was tested, so the numbers in the README can be put into perspective.

**Comment by lovasoa** [comment]
Yes, I agree it would be a useful thing to do, and I would accept a PR adding  a column to the results table with the library versions.

At the moment, most library versions can be found from reading the logs of our CI builds: https://travis-ci.org/lovasoa/bad_json_parsers
